Parsers/ASimProcessEvent/test/SentinelOne_ASimProcessCreate_SchemaTest.csv

```diff
@@ -0,0 +1,85 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcDomainType]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [TargetUserId]"
+"(2) Info: Missing optional field [ActingProcessFileCompany]"
+"(2) Info: Missing optional field [ActingProcessFileDescription]"
+"(2) Info: Missing optional field [ActingProcessFileInternalName]"
+"(2) Info: Missing optional field [ActingProcessFileOriginalName]"
+"(2) Info: Missing optional field [ActingProcessFileProduct]"
+"(2) Info: Missing optional field [ActingProcessFileSize]"
+"(2) Info: Missing optional field [ActingProcessFileVersion]"
+"(2) Info: Missing optional field [ActingProcessFilename]"
+"(2) Info: Missing optional field [ActingProcessIMPHASH]"
+"(2) Info: Missing optional field [ActingProcessInjectedAddress]"
+"(2) Info: Missing optional field [ActingProcessIntegrityLevel]"
+"(2) Info: Missing optional field [ActingProcessIsHidden]"
+"(2) Info: Missing optional field [ActingProcessSHA512]"
+"(2) Info: Missing optional field [ActingProcessTokenElevation]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUserAadId]"
+"(2) Info: Missing optional field [ActorUserSid]"
+"(2) Info: Missing optional field [ActorUserType]"
+"(2) Info: Missing optional field [ActorUserUpn]"
+"(2) Info: Missing optional field [AdditionalFields]"
+"(2) Info: Missing optional field [DvcAction]"
+"(2) Info: Missing optional field [DvcDescription]"
```

Comment:
can we set this? [DvcDescription](https://learn.microsoft.com/en-us/azure/sentinel/normalization-common-fields#dvcdescription)

Reply:
We are not getting it from SentinelOne data.